iib/workers/tasks/build.py

```diff
@@ -786,6 +786,7 @@ def _prepare_request_for_build(
         'binary_image_resolved': binary_image_resolved,
         'bundle_mapping': bundle_mapping,
         'from_index_resolved': from_index_info['resolved_from_index'],
+        'index_image_resolved': None,
```

Comment:
We can not get the `index_image_resolved` value at this point or why is this `None`?

Reply:
Then why bother adding it here?

Reply:
because response should be consistent

Reply:
right, I'll remove it